src/axom/inlet/tests/inlet_LuaReader.cpp

```diff
@@ -88,6 +88,33 @@ TEST(inlet_LuaReader_getString, getInsideStrings)
   EXPECT_EQ(value, "TesT StrInG");
 }
 
+
+TEST(inlet_LuaReader, mixLevelTables)
```

Comment:
I tried to create a really dumbed down version to show at the LuaReader level but it didn't work. It's a good test anyways.